src/modules/types/science/biology/growth_plate/style.css

```diff
@@ -0,0 +1,18 @@
+.cell-bottom {
```

Comment:
Maybe you could use more specific class name to prevent any conflict with other code.
.well-plate-cell-bottom
.well-plate-cell-top
for example